pages/api/artists/[name].ts

```diff
@@ -2,79 +2,58 @@ import type { NextApiRequest, NextApiResponse } from 'next'
 import { Prisma } from '@prisma/client'
 import prisma from '../../../lib/prisma'
 
-export const getArtistWithUserAndWorkAndLinks = (artistName: any) => {
+export const getArtistWithUserAndInviterAndLinks = (artistName: string) => {
   return prisma?.artist.findUnique({
     where: {
       handle: artistName
     },
     select: {
-      user: {
-        select: {
-          name: true,
-          email: true,
-          nominatedBy: {
-            select: {
-              name: true,
-              artist: {
-                select: {
-                  handle: true,
-                }
-              }
-            }
-          },
-        }
-      },
+      bio: true,
       pronouns: true,
       handle: true,
       title: true,
       mediums: true,
       mediumsOfInterest: true,
       location: true,
-      work: true,
       links: true,
-    },
-  })
-}
-
-export type ArtistWithUserAndWorkAndLinks = Prisma.PromiseReturnType<typeof getArtistWithUserAndWorkAndLinks>
-
-export const getArtistWithUserAndNominatedByAndWorkAndLinks = (name: string) => {
-  return prisma?.artist.findUnique({
-    where: {
-      handle: name,
-    },
-    include: {
       user: {
-        include: {
-          nominatedBy: {
-            include: {
-              artist: {
+        select: {
+          name: true,
+          email: true,
+          application: {
+            select: {
+              invitation: {
                 select: {
-                  handle: true
+                  inviter: {
+                    select: {
+                      name: true,
+                      artist: {
+                        select: {
+                          handle: true
```

Comment:
What's this query retrieving?

Reply:
This query grabs the entirety of the data used in displaying an Artist's page. The generated type is:
```(alias) type ArtistWithInviterAndUserAndLinks = {
    bio: string;
    pronouns: string;
    handle: string;
    title: string;
    mediums: string[];
    mediumsOfInterest: string[];
    location: string | null;
    links: Link[];
    user: {
        name: string;
        email: string;
        application: {
            invitation: {
                inviter: {
                    name: string,
                    artist: {
                        handle: string
                    },
                },
            },
        } | null;
    };
} | null```

Reply:
does that mean 
```
                    artist: {
                        handle: string
                    },
```
is optional? Since not all inviters are artists? but they are definitely users, so we would be able to get their name.